P5/Source/Specs/event.xml

```diff
@@ -85,7 +139,67 @@
       </person>
     </egXML>
   </exemplum>
+  <exemplum xml:lang="de-AT">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <listEvent type="generated">
+        <event xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001" ref="https://mrp.oeaw.ac.at/pages/show.html?directory=editions&amp;document=MRP-1-1-01-0-18480401-P-0001.xml" 
+          type="session">
+          <head>Nr. 1 Ministerrat, Wien, 1. April 1848</head>
+          <label>Ministerratssitzung <date when="1848-04-01">1848-04-01</date>, aus Band 1 1/1, März 1848–21. November 1848</label>
+          <listEvent type="agenda_items">
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_1">
+              <label>I. Enthebung Baron Karl Friedrich v. Kübecks als Finanzminister und Ernennung des Freiherrn Philipp v. Krauß zu seinem Nachfolger</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_2">
+              <label>II. Ernennung FML. Peter Zaninis zum Kriegsminister</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_3">
+              <label>III. Öffentliche Erklärung über die konstitutionellen Grundsätze; Umformung des Staatsrates</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_4">
+              <label>IV. Kriegserklärung Sardiniens; Vorgehen Radetzkys; Bestellung eines Hofkommissärs für Lombardo-Venetien</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_5">
+              <label>V. Beeidigung der Minister</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_6">
+              <label>VI. Tumulte im Kärntnertortheater</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_7">
+              <label>VII. Bewegung der tschechischen demokratischen Partei; Berufung Paul Josef Safaiřiks nach Wien</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_8">
+              <label>VIII. Behandlung der aus Lombardo-Venetien vertriebenen Justiz- und politischen Beamten</label>
+            </event>
+            <event type="agenda_item" xml:id="mrp-events-MRP-1-1-01-0-18480401-P-0001-top_1_1_1_9">
+              <label>IX. Besetzung der Landeschef und Ständepräsidentenposten in Prag; böhmische Deputation in Wien</label>
```

Comment:
Shouldn't it be:
"Landeschef- und Ständepräsidentenposten" meaning "Landeschefposten und Ständepräsidentenposten" ?

Reply:
Thanks @sabineseifert, you just discovered an OCR error that has not been postcorrected by our contractor, cf. https://hw.oeaw.ac.at/ministerrat/serie-1/a1-pdf.xml#a1-1848_z1.pdf. Will fix this in our retro digitized edition data.